docs/youtube/generic.md

```diff
@@ -22,7 +22,7 @@ Join our inclusive open source community EddieHub now for FREE at https://www.ed
 
 3. Open The Issue
 
-4. You get an invitation to the community by our GitHub Action 🎉
+4. You get an invitation to the community by our GitHub Action :tada:
```

Comment:
Not sure if YouTube can display emojis like this.

Reply:
@2802AdityA Please post a screenshot if this code displays emoji especially in the YouTube part as said by @schmelto.

Reply:
It is working on my local server  

![Screenshot from 2022-08-29 19-21-02](https://user-images.githubusercontent.com/84802479/187217276-cdad1f6b-6d4d-4870-ac4c-4b73aed3882b.jpg)

Reply:
Ok @2802AdityA.

Reply:
@schmelto @2802AdityA I have checked and found that YouTube does not render those code and it shows the text only. Please keep the emojis only and don't change it. It will not work.

Reply:
@schmelto @SaptarshiSarkar12  I have changed :tada: back to 🎉  in the Youtube file

Reply:
Great @2802AdityA !
@schmelto, the code looks good to merge.